bundle/edu.gemini.auxfile.workflow/src/main/java/edu/gemini/auxfile/workflow/CheckTask.java

```diff
@@ -17,7 +17,11 @@ public CheckTask(SPProgramID progId, File file, boolean checked) {
 	@Override
 	public void execute(Workflow wf) throws Exception {
         final AuxFileType type = AuxFileType.getFileType(getFile());
-        wf.getMailer().notifyChecked(getProgId(), type, getFile().getName(), _checked);
+
+        // Try to send out emails if notification for this file type is desired.
+        if (type.sendNotification()) {
```

Comment:
I think there might be a spaces/tabs issue in the formatting.

`sendNotification` seems fine to me too.  It boils down to a call `javax.mail.Transport.send` which is synchronous.  The `CheckTask` itself is running in a separate worker thread though.  At any rate, none of this behavior is new in the PR it's just that the new auxfile type apparently doesn't need to trigger the email so the flag had to be added to differentiate auxfiles that do and do not trigger notifications.